example/docusaurus.config.js

```diff
@@ -24,7 +24,7 @@ require('dotenv').config();
 					docs: {
 						sidebarPath: require.resolve('./sidebars.js'),
 						// Please change this to your repo.
-						editUrl: 'https://github.com/facebook/docusaurus/edit/main/website/',
+						editUrl: 'https://github.com/advanced-cropper/react-advanced-cropper/tree/master/example/',
```

Comment:
Not yet able to test and verify this fixes the `Edit this page` link but it seems correct and works if a simple replace operation is performed on the redirect url, using the old and new editUrl values